compiler/hash-ast/src/lib.rs

```diff
@@ -1,9 +1,17 @@
 //! Hash Compiler AST library file
 //!
-//! All rights reserved 2021 (c) The Hash Language authors
+//! All rights reserved 2022 (c) The Hash Language authors
 #![feature(generic_associated_types)]
 #![feature(iter_intersperse)]
 
+use hash_alloc::Castle;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    /// AST static allocator
+    pub static ref STATIC_CASTLE: Castle = Castle::new();
+}
+
```

Comment:
Eventually I expect we want to remove this?

Reply:
Yes definitely